k8s/resolver.py

```diff
@@ -57,65 +61,84 @@
 
 _THREADS = 32
 
-
+def _read_stamp_files(stamp_info_files):
+  """Reads the content from each of the files and appends the contents"""
+  stamps = ""
+  for filepath in stamp_info_files:
+    with open(filepath) as f:
+      content = f.read()
+      stamps += content
+
+  return stamps
+
+def _generate_stamp_info(stamp_list):
+  """Parses the string extracting out the stamp variables and values"""
+  stamp_info = {}
+
+  for stamp in stamp_list.splitlines():
+    key, value = stamp.split(" ")
+    stamp_info[key] = value
```

Comment:
should we check there are no duplicates as in https://github.com/bazelbuild/rules_docker/blob/master/container/join_layers.py#L240?

Reply:
Yeah, I think it make sense. I will add it.